### ISSUE TITLE
loss to convergence

### ISSUE THREAD

**Issue by 980202006** [mimbres/neural-audio-fp#15]
how much the loss when the model convergence

**Comment by mimbres** [comment]
It depends. Usually **train_loss = 0.2x** with the **tau = 0.05, 640_lamb** config file. 
![640_tau005_loss](https://user-images.githubusercontent.com/26891722/131213146-1d2561a2-1589-484f-84fa-7eeefb3e92e2.png)
Val_loss(red), Train_loss(blue).

On each epoch end, we do a mini-search-test. I think accuracy@1s and @3s with the g(f) can be a better way to validate.
![640_tau005_mini_test](https://user-images.githubusercontent.com/26891722/131213424-ba5a59c6-873a-43c9-ab32-b499f38cc954.png)
The value is about 82.x@1s and 94.x@3s.

**Comment by mimbres** [comment]
The learning curve is from the pre-trained model in #10 .

**Comment by 980202006** [comment]
thank you!

**Comment by mimbres** [comment]
The `loss` and `val_acc` is updated. Please refer https://github.com/mimbres/neural-audio-fp/issues/26#issuecomment-1122516028
